modules/pixi/PixiLayerOsm.js

```diff
@@ -128,6 +150,33 @@ export class PixiLayerOsm extends PixiLayer {
         }
       });
 
+
+      //Instructions to save 'canned' entity data for use in the renderer test suite:
+      //Set a breakpoint at the next line, then modify `this._saveCannedData` to be 'true'
+      //continuing will fire off the download of the data into a file called 'canned_data.json'.
+      //move the data into the test/spec/renderer directory.
+      if (this._saveCannedData && !this._alreadyDownloaded) {
+        const map = context.map();
+        const [lng, lat] = map.center();
+
+        let viewData = {
+          'lng': lng,
+          'lat': lat,
+          'zoom': zoom,
```

Comment:
One thing I was worried about, but is probably ok is:  this `zoom` isn't the real zoom, and this `projection` isn't the real projection. 

I _think_ that's ok because the purpose of this is really to setup the "arguments that the `osmLayer` functions will get called with" later when the benchmarking happens.

But the code being benchmarked might also depend on state in `context` or other places, so I'm not totally sure.  I think these specific benchmarks might be ok, but it's something to watch out for (the difference between pixi projection vs context projection, effective zoom vs map zoom, etc)